oneapp-server/dao/LocationDao.js

```diff
@@ -0,0 +1,22 @@
+const { SQLDataSource } = require('datasource-sql');
+
+class LocationDao extends SQLDataSource {
+  async getCounties() {
+    const query = this.knex.column('COUNTY_NUMBER', 'COUNTY_NAME')
+      .select()
+      .from('OA_COUNTY');
+    return query.cache(60 * 60);
```

Comment:
I like this for simple select statements to do the query directly instead of using the stored procedure. I reviewed the the `OA_PKG_GEN.SP_GET_COUNTIES` stored procedure, and this query covers everything in the stored procedure. 👍 

Bonus: Not really needed for a simple query like this, but so you're aware, I build a method to extract the list of requested fields from the GraphQL query. The list can then be passed into the `.column` method to dynamically return only the requested fields from the database. You can see an example of it here: https://github.com/newjersey/dhs-oneapp/blob/main/oneapp-server/types/Translation.js#L27